client/app/components/Header/index.jsx

```diff
@@ -112,7 +111,7 @@ export class Header extends React.Component<Props, State> {
     return (
       <div id="headerMobile" className={css.headerMobileNav}>
         <div>
-          {profile ? <HeaderProfile profile={profile} /> : null}
+          {profile ? <Avatar profile={profile} /> : null}
```

Comment:
Unfortunately, you can't swap this component out for `<Avatar />` since `<HeaderProfile />` has is more than just `<Avatar />`. The issue with the circular dependency comes from `import type { Profile } from './index';` in the HeaderProfile component.

Reply:
thank you for the help!
I'm going to continue reviewing this tomorrow.

Reply:
Oh you're right! The issue is with `import css from './Header.scss';` cause it's imported in both files!

Reply:
cool, just double checking
thanks!